pages/post/[slug].js

```diff
@@ -1,17 +1,26 @@
 import { useRouter } from "next/router";
 import { useDispatch, useSelector } from "react-redux";
 import { fetchPostBySlug } from "../../store/features/postSlice";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { useState } from "react";
+import { faClock } from "@fortawesome/free-regular-svg-icons";
+import { formateDate } from "../../utils";
 import md from "markdown-it";
 import Image from "next/image";
 import Avatar from "../../assets/images/user.png";
 import Script from "next/script";
 import config from "../../config";
 
 export default function Post() {
+  const [loaded, setLoaded] = useState(false);
   // get slug from URL
```

Comment:
remove comments from everywhere,if not require.

Reply:
it includes the behaviour of code,I will remove it when it is no longer needed.